include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -19,6 +21,17 @@ using namespace motor_messages;
  *
  */
 
+enum class ControlState {
+    FORCE_CONTROLLING,
+    POSITION_CONTROLLING,
+    IDLE,
+    ACTIVE
+};
+
+// TODO Investigate this value, I'm pretty sure that the timer actually gets
```

Comment:
Oh you're absolutely right about this. I forgot to factor in the period!

We probably want to shorten this hold-off period a bit.